pkgs/tools/security/bloodhound-python/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, python3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "bloodhound-python";
```

This should be called "bloodhound" as the package on PyPI is [called that way](https://pypi.org/project/bloodhound/) or `bloodhound-py` to avoid a conflict with [BloodHound](https://github.com/BloodHoundAD/BloodHound) which will most like be available in Nixpkgs soon as well.

```suggestion
  pname = "bloodhound-py";
```